osu.Game.Rulesets.Osu/Replays/OsuAutoGenerator.cs

```diff
@@ -229,9 +230,12 @@ private void moveToHitObject(OsuHitObject h, Vector2 targetPos, Easing easing)
         // Add frames to click the hitobject
         private void addHitObjectClickFrames(OsuHitObject h, Vector2 startPosition, float spinnerDirection)
         {
+            bool taiko = h.HitActions.Length == 1;
```

Comment:
obviously this can't exist.

Reply:
I don't understand what you mean. Is it that you just don't like that particular implementation (and in that case may I ask exactly why).